common/client-libs/validator-client/src/nymd/fee_helpers.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::nymd::GasPrice;
+use cosmos_sdk::tx::{Fee, Gas};
+use cosmos_sdk::Coin;
+use cosmwasm_std::Uint128;
+
+#[derive(Debug, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
+pub enum Operation {
+    Upload,
+    Init,
+    Migrate,
+    ChangeAdmin,
+    Send,
+
+    BondMixnode,
+}
+
+pub(crate) fn calculate_fee(gas_price: &GasPrice, gas_limit: Gas) -> Coin {
```

Comment:
Maybe it's a naming convention that I'm not familiar with, in which case just ignore this comment, but isn't it more of a `gas_fee` rather than a `gas_limit`? Same goes for every appearance of this naming in this PR.